python/cugraph/cugraph/tests/conftest.py

```diff
@@ -36,11 +36,11 @@ def dask_client():
         enable_tcp_over_ucx=True,
         enable_nvlink=True,
         enable_infiniband=True,
-        enable_rdmacm=True,
+        enable_rdmacm=False,
         # net_devices="mlx5_0:1",
```

Comment:
I suggest setting everything to `None`:
I think this not aligned with the current conftest

https://github.com/rapidsai/cugraph/blob/6c89b6359f853d9558d72c47d98e16615e3b329a/python/cugraph/cugraph/testing/mg_utils.py#L26-L28


You will have to make align it here.

Reply:
I believe this was resolved when I fetched the latest changes